thunder/executors/torchex.py

```diff
@@ -553,6 +554,7 @@ def _squeeze_transform(a: TensorLike, /, dim: None | int | Sequence[int] = None)
 _register_implementation(ltorch.tensor_split, tensor_split, checker=_always_executable)
 _register_implementation(ltorch.transpose, transpose, checker=_always_executable)
 _register_implementation(ltorch.unbind, unbind, checker=_always_executable)
+_register_implementation(ltorch.unfold, unfold, checker=_always_executable)
 _register_implementation(ltorch.unsqueeze, unsqueeze, checker=_always_executable)
```

Comment:
Are we not missing implementations for clang/prim symbols?

Reply:
Yupp sorry my bad!

Reply:
Let's test against `clang.unfold` then? Just to be safe

Reply:
You mean something like this?

``` python
In [1]: import torch; import thunder

In [2]: a = torch.arange(9)

In [3]: a
Out[3]: tensor([0, 1, 2, 3, 4, 5, 6, 7, 8])

In [4]: def f(x):
   ...:     return thunder.clang.unfold(x, 0, 2, 2)
   ...: 

In [5]: jf = thunder.jit(f, only_execute_prims=True)

In [6]: jf(a)
Out[6]: 
tensor([[0, 1],
        [2, 3],
        [4, 5],
        [6, 7]])
```

Reply:
I mean the op that we test in OpInfos is probably safer to change from `ltorch.unfold` to `clang.unfold`.

Reply:
Ohh I see! Is there a reason why sometimes in the tests we test against ltorch instead of clang?

Reply:
Not all ops in ltroch are in clang. And we still have a gap in testing decompositions unless the backward rule is implemented...